src/app/page.tsx

```
@@ -1,60 +1,247 @@
+          <div
+            className={`w-24 h-24 sm:w-32 sm:h-32 md:w-40 md:h-40 mx-auto mb-8 md:mb-12 rounded-full flex items-center justify-center ${
+              theme === "dark"
+                ? "bg-gradient-to-br from-orange-600 to-red-800 shadow-2xl shadow-orange-500/20"
+                : "bg-gradient-to-br from-orange-500 to-red-700 shadow-2xl shadow-orange-300/30"
+            }`}
+          >
+            <svg
+              className="w-12 h-12 sm:w-16 sm:h-16 md:w-20 md:h-20 text-white"
+              fill="currentColor"
+              viewBox="0 0 24 24"
+            >
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)